examples/b2c/checkout/integrations/classes/B2CCheckInventorySample.cls

```diff
@@ -1,6 +1,10 @@
 // This must implement the sfdc_checkout.CartInventoryValidation interface
 // in order to be processed by the checkout flow and used for your Check Inventory integration.
 global class B2CCheckInventorySample implements sfdc_checkout.CartInventoryValidation {
+    // To access the service below, you may need to add https://b2b-commerce-test.herokuapp.com in Setup | Security | Remote site settings.
+    private static String httpHost = 'https://b2b-commerce-test.herokuapp.com';
```

Comment:
This url should not be a working url and instead should be updated to

http://example.com/path